### PR TITLE
That new new

### PR DESCRIPTION
**Pull request by fabianperez**
ayyyy

got some basic shit together for our lil corner of the internet with ~features~ like:
- dj/vj roster
- functional newsletter sign up
- Social Media
- latest poster

image

stuff i need from you:
- a :+1: (or :-1:) on this
- your soundcloud, mixcloud, or wherever you keep your mixes / videos if you want to get linked up. 
- your love and affection

stuff i want from you:
- suggestions on copy. i'm basic :information_desk_person: 
- add yourself to this if you aren't on the list somewhere. sorry if i've missed you

/cc @ZeroFucks/humans

### CODE REVIEW

**Comment by thedaniel** [comment]
:heart_eyes_cat:

I threw in links to mixcloud accts for all the DJs but fabian as a start - change as needed

**Comment by rsanheim** [comment]
This is pretty hot.

@amandelman you should add yourself to the html here with whatever dj name / link you want.

File: index.html [hunk 1]
```diff
@@ -10,8 +9,74 @@
...
+          <section>
+            <h3 class="section-header">Zero Fucks is</h3>
+            <ul class="roster">
+              <li><a href="https://www.mixcloud.com/atmosdotorg/">atmøs</a> <span class="human-meta">SF</span></li>
+              <li><a href="https://www.mixcloud.com/dpeshmo/">dpeshMo</a> <span class="human-meta">SF</span></li>
+              <li>fabian <span class="human-meta">SF</span></li>
+              <li><a href="https://www.mixcloud.com/rsanheim/">sanheim</a> <span class="human-meta">BERLIN</span></li>
+              <li><a href="https://www.mixcloud.com/thedaniel/">Rationale</a> <span class="human-meta">LA &middot; <a href="https://soundcloud.com/therationale">Originals</a></span></li>
+              <li><a href="https://www.mixcloud.com/djspacecamp/">DJ Spacecamp</a> <span class="human-meta">SF</span></li>
+              <li><a href="http://www.mixcloud.com/Surf_n_Turf/">Surf 'n' Turf</a> <span class="human-meta">Black Rock City</span></li>
```
Comment (rsanheim) [inline index.html:43]: :+1: :dancers:

**Comment by amandelman** [comment]
Added Surf 'n' Turf w/ mixcloud link

**Comment by rsanheim** [comment]
Got Surf n Turf on there...looks good from the Madison side of the world.

**Comment by thedaniel** [comment]
lol i better add cease & desist then

**Comment by thedaniel** [comment]
I feel like this can be merged once @fabianperez links up his own damn name, people can update their links to be not mixcloud if they want whenever

**Comment by fabianperez** [comment]
finna merge

**Pull request merged**